remote_fun/Makefile

```diff
@@ -0,0 +1,13 @@
+.PHONY: compile run
+
+default: run
+
+compile:
+	erl -compile remote_fun_server
```

Comment:
[erlc](http://erlang.org/doc/man/erlc.html) is more frequently used to compile from the shell. Also usually you'll have a `src` module with the .erl files and an `ebin` module where you put the compiled ones.

In any case it's easier to stick to rebar3 as stated below.